frontend/css/year-in-music.less

```diff
@@ -568,7 +568,7 @@
       overflow: hidden;
       .graph {
         color: var(--selectedColor);
-        overflow: auto;
+        overflow: scroll;
```

Comment:
This looks better IMO, and doesn't hide the tooltips:
```suggestion
        overflow: visible;
```

Before:
![image](https://github.com/user-attachments/assets/2d005aad-962d-4d10-8cb7-1645decf71bd)


After:
![image](https://github.com/user-attachments/assets/390b134c-5e3c-4078-8d67-eb05969f9bc2)

Reply:
If we set overflow to visible, it breaks on smaller screens 

<img width="370" alt="image" src="https://github.com/user-attachments/assets/e6545032-72c7-4b0d-aa86-175b17159ed6">

Reply:
Maybe`overflow-y: hidden;` then, I think that resolves both issues

Reply:
Yep! It does the job